mmdet/integration/nncf/compression.py

```diff
@@ -0,0 +1,165 @@
+import pathlib
+import torch
+
+from mmdet.utils import get_root_logger
+from .utils import (check_nncf_is_enabled, get_nncf_version, is_nncf_enabled,
+                    load_checkpoint)
+
+if is_nncf_enabled():
+    try:
+        from nncf import (NNCFConfig, create_compressed_model,
+                          register_default_init_args)
+        from nncf.dynamic_graph.patch_pytorch import nncf_model_input
+        from nncf.initialization import InitializingDataLoader
+        from nncf.nncf_network import NNCFNetwork
+
+        class_InitializingDataLoader = InitializingDataLoader
+    except:  # noqa: E722
```

Comment:
is it hard to determine exact exception class?

Reply:
It is `ImportError`.

Fixed in one of my last commits.